src/main/java/frc/utils/ComputerLogFile.java

```diff
@@ -0,0 +1,88 @@
+package frc.utils;
+
+import frc.robot.constants.DirectoryPathsConstants;
+
+import java.awt.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+
+public class ComputerLogFile {
+
+    private static final String TYPE_OF_FILE = ".txt";
+
+    private static final Path LOG_FILES_ACTIVITY_FILE_PATH = new ComputerLogFile("ComputerLogFile").path;
```

Comment:
add a comment and consider changing the "ComputerLogFile" to "ActivityFilesLogFile" or hopefully someting better